form-editor-test-data/content/Sites/Demo/Germany/German/Content/Articles/Einverstanden.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Ich denke es ist einheitlicher, wenn die Testdokumente komplett in englisch angelegt werden. Zusätzlich würde ich die xml-Dokumente noch einmal automatisch formatieren lassen, sodass die Einrückung bei allen Dokumenten stimmt.